hack/jenkins/e2e.sh

```diff
@@ -534,6 +534,7 @@ case ${JOB_NAME} in
           ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
           ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
           ${TRUSTY_SKIP_TESTS[@]:+${TRUSTY_SKIP_TESTS[@]}} \
+          "Monitoring\sshould\sverify\smonitoring\spods\sand\sall\scluster\snodes\sare\savailable\son\sinfluxdb\susing\sheapster" \
```

Comment:
> I don't think we should have it here - this should be moved to TRUSTY_SKIP_TESTS

We do. :) ```TRUSTY_SKIP_TESTS``` are skipped also by Trusty-beta jobs but this test would pass and should be run with Trusty-beta (which does not have systemd, which was the origin of this flakiness)